CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 - A memory bug about offsets calculation has been solved; besides, the `calculate_offsets` function has been sped-up by a significant amount.
 
 # v5.8
+- New function `remove_all!` replaces `genocide!` (now deprecated).
```

Comment:
Put these new changes in `main`

Reply:
I hope my "request changes" doesn't seem too much like I'm bossing you around @MA-Ramirez! I don't like telling people what to do when I'm not super confident that it's the right thing, but I'm pretty sure `main` is the right place for these changes :')